schedule/migration/aarch64_regression_test_online.yaml

```diff
@@ -60,57 +55,15 @@ conditional_schedule:
         - migration/online_migration/register_system
         - migration/online_migration/zypper_patch
         - installation/install_service
-  openldap_to_389ds:
-    REGRESSION_389DS:
-      1:
-        - migration/openldap_to_389ds
   regression_tests:
     REGRESSION_TEST:
       1:
         - locale/keymap_or_locale
-        - console/supportutils
-        - console/force_scheduled_tasks
```

Comment:
`force_scheduled_tasks` is a basic one to ensure the load is ok, sometimes the system is too busy to actually do anything and it is recommended to schedule it.